components/blog/Authors.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <b-row
+    class="flex-md-row pb-3"
+    data-qa="blog author"
+  >
+    <b-col cols="12">
+      <h3 class="font-weight-bold">
+        #{{ authorTitle }}
+      </h3>
+      <BlogAuthor
+        v-for="(author, index) in authors"
+        :key="index"
+        :name="author.fields.name"
+        :organisation="author.fields.affiliation"
+        :url="author.fields.url"
+      />
+    </b-col>
+  </b-row>
+</template>
+
+<script>
+  import BlogAuthor from '../blog/Author';
+
+  export default {
+    name: 'Authors',
```

Comment:
[Component names should always be multi-word.](https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential)